### DISCUSSION TITLE
Can someone that has an Arxiv account send me Crawford's email?

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#241] [Astrophysics and Physics]
It is existential for all non-expandist cosmologists in this forum to prove that cosmic time dilation does not exists.
Until recently, quasars and supernovaes were shown to be devoid of cosmic time dilation, however for SNies this has recently been questioned by a DES study.
As such it is existential to perform a better analysis with the highest redshift supernovaes otherwise static universes would be simply 100% falsified.  You might not realize it, but this is the highest priority issue in non-orthodox cosmology.
To help with this quest, I need to contact Dr Crawford,
one can get his email via the button "view email" here
https://arxiv.org/abs/1711.11237
https://arxiv.org/show-email/b3a46a20/1711.11237
However since I do not have an arxiv account I cannot get it.
Note that Crawford is retired but he could still give insightful answers, moreover he recently (2 years ago?) signed a petition with members of the ACG group about arxiv censorship.
Thanks in advance, I promise I will be respectful of his time and not spam him with too many questions.

**Comment by LifeIsStrange** [comment]
Edit: I have made an arxiv account and was able to get his email, nevermind

**Comment by mikehelland** [comment]
This is a pretty low effort comment on my part, but he seems to be accused of using data translated to the rest-frame, where time dilation would already be removed:

https://www.physicsforums.com/threads/lack-of-time-dilation-in-type-1a-supernova.946267/

**Reply by LifeIsStrange** [reply]
Crawford is literally the one pointing out SALT2 rest frame remove the info.
His analysis of raw light curves and absolute magnitudes bypass this.

**Reply by mikehelland** [reply]
Do you have a link to the raw data?

**Reply by LifeIsStrange** [reply]
It is detailed in section 2 of the paper
2.2 Raw type Ia supernovae light curves 
The dataset is B14
Crawford detail each Metadata associated with a supernova, you can also see the files table 1
> All of the original SNe observations were retrieved from the SNANA [17] website using the index files shown in Table 1.

So I'm not sure I understand 
The paper for the B14 dataset compilation (2011)
https://arxiv.org/abs/1104.1443
Link to
https://tspace.library.utoronto.ca/handle/1807/25390
While it contains SNLS I'm not sure where is the link for the full B14 dataset
Thankfully the files Crawford used are explicitly stated table 1

But then he says he used the data from the "SNANA" website which is from SDSS.
https://arxiv.org/abs/0908.4274

Link to SNANA
http://www.sdss.org/supernova/SNANA.html
And the link is dead yay
But it might be available on webarchive? 
There is a classic version of the sdss website 
https://classic.sdss.org/supernova/aboutsupernova.php
It mentions snana
And gives another SNANA link which is also dead 
http://snana.uchicago.edu/
So SNANA is a software package as discussed on sncosmo
https://github.com/sncosmo/sncosmo/issues/197
And indeed here is the new SNANA repository 
https://github.com/RickKessler/SNANA
I suppose the software allows to download the files of table 1.
Datasets seems to be available here for snana
https://zenodo.org/record/4015325

Here is the installation guide 
https://github.com/RickKessler/SNANA/blob/master/doc/snana_install.pdf

To use a curve fitter like SALT2 there is also the sncosmo python library.

**Reply by mikehelland** [reply]
Trying to compile the dataset Crawford used sounds tedious, and is a low priority for me at the moment. Since you have his email, perhaps you can request it directly from him.

*eta* I'm downloading [SNDATA_ROOT-PUBLIC-2020-07-11.tar.gz](https://zenodo.org/records/4015325/files/SNDATA_ROOT-PUBLIC-2020-07-11.tar.gz?download=1) now. But I think that's only one part of his full dataset.

**Reply by LifeIsStrange** [reply]
Even if you did not manage to find the B14 full dataset, by far the most important subdataset of light curves is SNLS.
Which is probably this
https://tspace.library.utoronto.ca/handle/1807/26549

For processing the light curves, the sncosmo python package should be the most user friendly even though Crawford used SNANA.

The reason why SNLS is most important is because it goes to high redshift and has a decent number of supernovae (Cf table 1), the hubble ones are low numbered and the sdss ones are low redshift.

The main goal would be to reproduce figure 2 of Crawford paper and also to a lower extent figure 3.

If you could that would be groundbreaking as, as I said previously cosmic time dilation is the biggest crisis in cosmology by far.

I will send him an email if we don't manage to find the data of table 1 after a few days.

**Reply by ExpEarth** [reply]
David Crawford has a profile in ResearchGate.  There is a listing of some of his papers there and his email.

https://www.researchgate.net/profile/David-Crawford-6

**Reply by LifeIsStrange** [reply]
I found his email but thanks yes it's also possible to DM him on researchgate.

**Reply by mikehelland** [reply]
>If you could that would be groundbreaking as, as I said previously cosmic time dilation is the biggest crisis in cosmology by far.

The video I posted used observed frame data, and there's also this:

https://arxiv.org/abs/astro-ph/0504481

Which has no mention of SALT.

**Reply by LifeIsStrange** [reply]
So I downloaded SNDATA ROOT PUBLIC and indeed all the light curves files of table 1 are listed under the lcmerge folder
e.g. JLA2014_SNLS.LIST
Now the question is how to process .LIST files

**Reply by mikehelland** [reply]
That LIST file is just a text document that contains the file names of the rest of the directory:

    JLA2014_SNLS_03D1au.dat
    JLA2014_SNLS_03D1aw.dat
    JLA2014_SNLS_03D1ax.dat
    JLA2014_SNLS_03D1bp.dat
    JLA2014_SNLS_03D1co.dat
    JLA2014_SNLS_03D1dt.dat
    JLA2014_SNLS_03D1ew.dat
    JLA2014_SNLS_03D1fc.dat
    JLA2014_SNLS_03D1fq.dat
    JLA2014_SNLS_03D3aw.dat
    JLA2014_SNLS_03D3ay.dat
    JLA2014_SNLS_03D3ba.dat
    JLA2014_SNLS_03D3bl.dat
    ...

The actual goodies are in the .dat files.

The .dat files have a MJD, that's your julian date. A difference of 1 between records should be 1 day.
